test/functional/api/cas/casadm.py

```diff
@@ -61,6 +64,15 @@ def add_core(cache: Cache, core_dev: Device, core_id: int = None, shortcut: bool
     return core
 
 
+def add_lvm_as_core(cache: Cache, lvm: Lvm):
```

Comment:
As discussed with Kasia, this method will be removed and I will modify 'add_core' method to identify and use LVM as core.